### PR TITLE
Align styles with current brand

### PR DESCRIPTION
**Pull request by rasmusguterstam**
A long time coming on this one: get rid of the double sign-in, and align the OAuth Gatekeeper prompt to current brand 💅

https://github.com/HedvigInsurance/hedgehog/pull/288

### CODE REVIEW

**Review by vonElfvin — APPROVED** [review]
😮

**Review by palmenhq — CHANGES_REQUESTED** [review]
Great stuff! However shall we update the other templates also?

Or - even better - refactor a bit. Feels like the templates have a lot of duplicate code. Maybe we could utilise [template extensions](https://pebbletemplates.io/wiki/tag/extends/)?

File: src/main/resources/templates/sso/base.html.peb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          </div>
+        </div>
+        {% block content %}
+        {% endblock %}
+      </div>
+</div>
+<script>
+{% block script %}
+{% endblock %}
+</script>
```
Comment (palmenhq) [inline src/main/resources/templates/sso/base.html.peb:49]: what about placing the script tags inside the block in the extensions? That way i think it's easier to know what to put here.

**Review by palmenhq — DISMISSED** [review]
Good stuff!

**Review by palmenhq — APPROVED** [review]

**Pull request merged**